libs/nss.c

```diff
@@ -15,6 +15,13 @@ static __thread DIR *tcbdir = NULL;
 
 int _nss_tcb_setspent(void)
 {
+	if (!tcbdir) {
+		tcbdir = opendir(TCB_DIR);
+		if (!tcbdir)
+			return NSS_STATUS_UNAVAIL;
+	}
+
+	rewinddir(tcbdir);
```

Comment:
What's the reason for doing `rewinddir` even right after successful `opendir`?

Reply:
No technical reason, just to keep the previous `if ` simple.

Reply:
I could insert another `return 1` after initialization to omit the call to `rewinddir`…

Reply:
Should be resolved now.

Reply:
The indentation of the added return statement looks wrong.

Reply:
> The indentation of the added return statement looks wrong.

Fixed in rebased commit.